src/main/kotlin/coverosR3z/server/ServerUtilities.kt

```diff
@@ -26,6 +27,12 @@ data class PreparedResponseData(val fileContents: String, val code: String, val
  */
 val pageExtractorRegex = "(GET|POST) /(.*) HTTP/1.1".toRegex()
 
+/**
+ * Used for extracting the length of the body, in POSTs and
+ * responses from servers
+ */
+val contentLengthRegex = "Content-Length: (.*)$".toRegex()
```

Comment:
Did this jump files?

Reply:
yes, may as well be on the SUT side rather than the test side